lib_static/redmine/i18n.rb

```diff
@@ -133,18 +129,25 @@ def format_time_as_date(time, format = nil)
       end
     end
 
-    def format_time(time, include_date = true)
+    def format_time(time, include_date: true, format: Setting.time_format)
```

Comment:
It's hard to know that this method uses user time zone without reading its code. We could rename it, but I propose adding some docs for it.

```suggestion
    # Formats the given time as a time string according to the user's time zone
    # and optional specified format.
    #
    # @param time [Time, String] The time to format. Can be a Time object or a
    #   String.
    # @param include_date [Boolean] Whether to include the date in the formatted
    #   output. Defaults to true.
    # @param format [String] The strftime format to use for the time. Defaults
    #   to the format in `Setting.time_format`.
    # @return [String, nil] The formatted time string, or nil if the time is not
    #   provided.
    def format_time(time, include_date: true, format: Setting.time_format)
```